src/content/docs/zh/build/smart-contracts/cryptography.mdx

```diff
@@ -102,7 +102,7 @@ Aptos 密码学扩展在 Move 中的设计和集成由三个基本原则指导:
 ## 椭圆曲线算术
 
 虽然 [哈希函数](#cryptographic-hash-functions) 和 [数字签名](#digital-signature-verification) 模块应为大多数应用提供足够的功能,但某些应用将需要更强大的密码学.
-通常,此类应用的开发者需要等待其所需的密码学功能在 [Aptos Move 框架](/zh/build/why-move) 中作为 [Move 原生函数](/zh/build/smart-contracts/book/functions#native-functions) 高效实现.
+通常,此类应用的开发者需要等待其所需的密码学功能在 [Aptos Move 框架](/zh/build/smart-contracts/why-move) 中作为 [Move 原生函数](/zh/build/smart-contracts/book/functions#native-functions) 高效实现.
```

Comment:
I think why move may be under network

Reply:
@gregnazario unless there is another one, this is what I have: https://aptos.dev/build/smart-contracts/why-move